tests/app/tests/test_webpack.py

```diff
@@ -334,8 +334,9 @@ def test_request_blocking(self):
             self.assertTrue(elapsed < wait_for)
 
     def test_skip_common_chunks_djangoengine(self):
-        'Test case for deduplication of modules with the django engine.'
+        """Test case for deduplication of modules with the django engine."""
```

Comment:
Not to be splitting hairs but the triple quotes are totally unnecessary. A docstring is a docstring either way.

Reply:
I agree it works either way. I don't have strong opinions about it.

Since we are using the `"""` in other places, I think it would be best to keep the same usage pattern.